dune-project

```diff
@@ -77,4 +77,11 @@
    (>= 3.11.0))
   (js_of_ocaml-toplevel
    (>= 3.11.0))
-  (alcotest :with-test)))
+  (alcotest :with-test)
+  ; hilite
+  textmate-language
+  ; tools/ood-gen
```

Comment:
These are listed in the Makefile in the `DEV_DEPS` variable. The reason for that is: they are not dependencies of the project, they are only used to re-generate the data modules. So typically, we don't want them to be installed in Docker.

Reply:
Perhaps these dependencies can be marked `{dev}` then.

Reply:
I think `with-test` makes more sense, I've updated the PR.

Reply:
Updated!